packages/apps-config/src/endpoints/testingRelayWestend.ts

```diff
@@ -81,6 +81,7 @@ export function createWestend (t: TFunction): EndpointOption {
       },
       {
         info: 'moonshadow',
+        isDisabled: true, // https://github.com/polkadot-js/apps/issues/6181
```

Comment:
```suggestion
        isUnreachable: true, // https://github.com/polkadot-js/apps/issues/6181
```

Reply:
parachain, keep unreachable (so it is still visible as an icon on the parachains page)